workitem/typegroup.go

```diff
@@ -95,6 +95,17 @@ func TypeGroups() []WorkItemTypeGroup {
 	}
 }
 
+// TypeGroupByName returns a type group based on its name if such a group
+// exists; otherwise nil is returned.
+func TypeGroupByName(name string) *WorkItemTypeGroup {
```

Comment:
If this function is called more often, we should consider altering the type group data structure. Probably an ordered map or map and array for order would be fine.

Reply:
This will come once we store WIT groups persistently.